### PR TITLE
[DX-798] - Update CTA links on home page badges to refer to docs pages

### PR DESCRIPTION
**Pull request by dcs3spp**
Update CTA links on home page badges to refer to docs pages

[DX-798](https://tyktech.atlassian.net/browse/DX-798)

[preview](https://deploy-preview-3542--tyk-docs.netlify.app/docs/nightly/)

[DX-798]: https://tyktech.atlassian.net/browse/DX-798?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by letzya — DISMISSED** [review]

**Comment by letzya** [comment]
@dcs3spp  it doesn't work if I click on the homepage i get the old urls ...

**Review by letzya — CHANGES_REQUESTED** [review]
@dcs3spp it doesn't work if I click on the homepage i get the old urls ...

**Comment by yurisasuke** [comment]
> @dcs3spp it doesn't work if I click on the homepage i get the old urls ...

@letzya  it is that @dcs3spp  has included the wrong preview link.It should be https://deploy-preview-3542--tyk-docs.netlify.app/docs/nightly/

**Comment by yurisasuke** [comment]
Looks good. However you should change preview link to https://deploy-preview-3542--tyk-docs.netlify.app/docs/nightly/

**Comment by dcs3spp** [comment]
> @dcs3spp it doesn't work if I click on the homepage i get the old urls ...

Thanks @Keithwachira. @letzya I have updated the preview link

File: tyk-docs/data/tyk_components.json [hunk 1]
```diff
@@ -5,23 +5,23 @@
...
       "steps": "Only 2 steps",
       "button_text": "Free trial",
-      "path": "tyk-cloud/",
+      "path": "deployment-and-operations/tyk-cloud-platform/quick-start/",
       "description":"A fully managed service that makes it easy for API teams to create, secure, publish and maintain APIs at any scale, anywhere in the world.",
       "includes": "Includes: Tyk API Gateway, Tyk Dashboard, Tyk Portal, Tyk UDG"
     },{
       "title": "Tyk Open Source",
       "image": "img/tyk_open_source.svg",
       "steps": "Only 4 steps",
```
Comment (letzya) [inline tyk-docs/data/tyk_components.json]:
```suggestion
      "steps": "Only 3 steps",
```

**Review by letzya — APPROVED** [review]
A small fix to the content - it's 3 steps for oss not 4

**Pull request merged**